ibm_concert/cert_renewal/roles/send-template/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
role likely not  needed if using templating feature of zos_job_submit.

Reply:
These roles were created in the early days of the samples to show new Ansible users on how to structure roles in playbooks. Would require rewrite and testing if we remove them now.